documentation/MeetingMinutes/MOM-2023-07-12.md

```diff
@@ -0,0 +1,115 @@
+<br>Jesús (Telefonica) replies that with existing flows in GSMA there is no need of having such API.
+<br>Mark (GSMA) clarifies that the repository is open for anyone to join.
+<br>Shilpa (DT) regarding components like Telco Finder, it was not in original scope. But AuthN/AuthZ was, so should be discussed there.
+<br>Gautam (SMI) indicates that AuthN/AuthZ is in scope of OIDF, so asks what CAMARA wants to define.
+<br>Shilpa (DT) and Jesus (Telefonica) clarifies that in any case CAMARA will not re-define standards or define new ones, but document how to use existing standards.
+<br>Gautam (SMI) sees an implicit relationship.
+<br>...
+
+<br><br>... Discussion on the topic continues, but it is agreed that AuthN/AuthZ is in the scope of CAMARA and that the discussion should continue in this subproject. As a way forward, it is agreed to fill [user-consent.md](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/user-consent.md) with the information related to the CAMARA APIs access flows approved in the GSMA. So that every CAMARA participant has the necessary context and can ask any questions. GSMA discussions and agreements would be shared in CAMARA as appropriate, and CAMARA discussions may feed back to the GSMA if an issue relevant to the GSMA is raised.
+
+<br>
+
+## Issues and PRs
+
+| **Issue** | Owner | Description |
+| ----------| ----- | ----------- |
+| **Open issues** |  |  | 
+| [#15](https://github.com/camaraproject/IdentityAndConsentManagement/issues/15) | GSMA/Vonage | **Subscriber Identity API**<br> Not covered in today's call. Time running out with previous discussions and agenda items. | 
+| [#21](https://github.com/camaraproject/IdentityAndConsentManagement/issues/21) ([PR#33](https://github.com/camaraproject/IdentityAndConsentManagement/pull/33))| Deutsche Telekom AG | **Definition of scenarios for consent management**<br> Not covered in today's call. |
+| [#32](https://github.com/camaraproject/IdentityAndConsentManagement/issues/32) | Orange | **The concept of purpose**<br> Sebastien (Orange) indicates that after analysis, they don’t see benefit in the use of purpose.<br>Discussion between Telefonica and Orange, regarding the implications. Telefonica states that scopes need not be changed if purposes are used.<br>Klaus (Vodafone) states that purpose may be different per country, so that’s a reason to have the concept separated<br>Proposal for way forward, as this issue is lasting long, is to have input on how to solve a practical scenario, using Orange proposal and using Telefonica proposal. In this way, the two solutions could be compared more easily and all participants could agree on the final solution (one of them or any other compromise solution in between that might be agreed).<br>Shilpa (DT) challenges whether there is a need to provide the purpose, if that is agreed in the onboarding phase. |
```

Comment:
I would propose to change the line "Shilpa (DT) challenges whether there is a need to provide the purpose, if that is agreed in the onboarding phase. " with
Shilpa (DT) raised the question regarding how the final legal consent is derived in a case where multiple purposes are agreed with the ASP and the different purposes could indicate a different legal basis. And if just a single purpose is agreed with the ASP at the time of onboarding, why do we need to pass it within an API request. 
Jesus (TEF) mentioned that in such cases where multiple purposes are involved, some kind of policy will have to be agreed to ensure that the user experience is not impacted and that we can derive the needed legal basis (please feel free to change your content if I have not interpreted it correctly).

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/51/commits/c4964a20255e6aa763227ec5d25d8d0e5cef033c